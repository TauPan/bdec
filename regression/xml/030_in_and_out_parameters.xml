<protocol>
    <!--
      Test what happens when we have a parameter that is to be passed out
      of an entry, but also into a child (issue122).
      
             ___ e ___
        __c__         d(len=a)
       a   b(len=a)
      -->
    <sequence name="e">
        <reference name="c" />
        <reference name="d" />
    </sequence>
    <common>
        <field name="a" length="8" type="integer" />
        <field name="b" length="(${a} * 8)" />
        <sequence name="c">
            <reference name="a" />
            <reference name="b" />
        </sequence>
        <field name="d" length="(${c.a} * 8)" />
    </common>
</protocol>
