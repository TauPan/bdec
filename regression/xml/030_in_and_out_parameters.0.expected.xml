
           <e>
             <c>
               <a>2</a>
               <b>01100001 01100001</b>
             </c>
             <d>01100010 01100010</d>
           </e>